exonum/src/api/node/public/system.rs

```diff
@@ -74,15 +76,17 @@ impl SystemApi {
     }
 
     fn handle_stats_info(self, name: &'static str, api_scope: &mut ServiceApiScope) -> Self {
+        let self_ = self.clone();
```

Comment:
I guess we could get rid of the clone. We just can get a `tx_cache_size` here and this value will move into the closure scope. Nope ?

Reply:
```rust
fn handle_stats_info(self, name: &'static str, api_scope: &mut ServiceApiScope) -> Self {
    let tx_cache_size = self.shared_api_state.tx_cache_size();
    api_scope.endpoint(name, move |state: &ServiceApiState, _query: ()| {
        let snapshot = state.snapshot();
        let schema = Schema::new(&snapshot);
        Ok(StatsInfo {
            tx_pool_size: schema.transactions_pool_len(),
            tx_count: schema.transactions_len(),
            tx_cache_size,
        })
    });
    self
}
```

Reply:
In your variant, we will obtain tx_cache_size only once and then use this captured value for all requests.

Reply:
Sure. My fault.